packages/ui/src/views/layouts/SandboxLayout.tsx

```diff
@@ -79,13 +83,12 @@ interface ContentProps {
   children: ReactNode
   maxWidth?: 'sm' | 'md' | 'lg' | 'xl' | '2xl' | '3xl' | '4xl' | '5xl' | '6xl' | '7xl'
   className?: string
-  paddingClassName?: string
 }
 
-function Content({ children, maxWidth, className, paddingClassName = 'px-5 pt-7 pb-11' }: ContentProps) {
-  const widthClass = maxWidth ? `max-w-${maxWidth} mx-auto` : ''
+function Content({ children, maxWidth, className }: ContentProps) {
+  const widthClass = !!maxWidth && `max-w-${maxWidth} mx-auto`
```

Comment:
`max-w-${maxWidth}` would be problematic. The tailwind utilities don't understand concatenation since it's at runtime and class-pruning is at build time. These classes will only exist if they are referenced in full somewhere the "safe" option would be to use the `maxWidth` as a look up to a map that brings back the full classname.

e.g.

```typescript
type maxWidthOptions = 'sm' | 'md' | 'lg' | 'xl' | '2xl' | '3xl' | '4xl' | '5xl' | '6xl' | '7xl'

const maxWidthMap: Record<maxWithOptions, string> = {
  sm: 'max-w-sm',
  md: 'max-w-md',
  // ...
  '7xl': 'max-w-7xl'
}

interface ContentProps {
  children: ReactNode
  maxWidth?: maxWidthOptions
  className?: string
}

// ...

`${maxWidthMap[maxWidth]} mx-auto`

Reply:
This way it works well - I tested in the app

<img width="617" alt="image" src="https://github.com/user-attachments/assets/95ef77fc-f8ba-4a95-aae3-4428c36ae73f" />

Reply:
We also use concatenation in the `LabelMarker` component, for instance

Reply:
By the way, your approach will also fail because classnames from `maxWidthMap` won't be added to the CSS bundle since they aren't used within the `className` prop. To fix this, you still need to add them to Tailwind's `safelist`

Reply:
Tailwind's use of CSS Prune used to be very open and just scanned for the text in any known file, regardless of how it was being used, weird that they'd make a change to specifically target `className` since that's a React-specific thing (all the other frameworks use just `class`) and the tailwind team has been famously framework agnostic 🤔 ¯\\_(ツ)_/¯

Reply:
I could probably mislead you here, so my statement needs to be double-checked😓
But I had this issue in a different project that is written in Go/Templ: once you take something from the `class` prop, Tailwind loses it. And `safelist` works well for cases like this